src/main/java/org/araymond/joal/core/bandwith/BandwidthDispatcher.java

```diff
@@ -169,12 +169,12 @@ void recomputeSpeeds() {
         for (final InfoHash infohash : this.torrentsSeedStats.keySet()) {
             this.speedMap.compute(infohash, (hash, speed) -> {
                 if (speed == null) {
-                    return new Speed(0);
+                    return new Speed();
```

Comment:
`return new Speed.ZERO`; rather than default value (see comment in class Speed.java)

Reply:
```suggestion
    return Speed.ZERO;
```

Reply:
If we were to use singleton `Speed.ZERO`, then Speed type would need to be made immutable; meaning we don't update Speed objects' values in `recomputeSpeeds()`, but create new instances with each update. Not an issue, but not sure I'd prefer this.